src/workflow/RunEnergyPlus.cpp

```diff
@@ -154,14 +154,30 @@ void OSWorkflow::runEnergyPlus() {
     // TODO: is this the right place /Do we want to do that if we chose epJSON?
     detailedTimeBlock("Saving IDF", [this, &inIDF] { workspace_->save(inIDF, true); });
 
+    std::ofstream stdout_ofs(openstudio::toString(runDirPath / "stdout-energyplus"), std::ofstream::trunc);
```

Comment:
Declare it higher, to trunc it first, because I'm going to pipe expand objects output to it as well.